### PR TITLE
adjust code style to PSR-2

### PR DESCRIPTION
**Pull request by thyseus**
hey,

i ran the code through a tool to make it PSR-2 compatible.

please consider this PR.

### CODE REVIEW

File: LanguageChangedEvent.php [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?php
+
```
Comment (mikehaertl) [inline LanguageChangedEvent.php]: I can't find this extra blank line in PSR-2 and the example there also don't use it. Can you explain?

File: UrlManager.php [hunk 2]
```diff
@@ -171,7 +172,7 @@ class UrlManager extends BaseUrlManager
     public function init()
     {
         if ($this->enableLocaleUrls && $this->languages) {
-            if (!$this->enablePrettyUrl) {
+            if (! $this->enablePrettyUrl) {
```
Comment (mikehaertl) [inline UrlManager.php]: Same here: Where is this mentioned in PSR-2?

File: UrlManager.php [hunk 3]
```diff
@@ -229,7 +230,7 @@ public function parseRequest($request)
     public function createUrl($params)
     {
         if ($this->ignoreLanguageUrlPatterns) {
-            $params = (array) $params;
+            $params = (array)$params;
```
Comment (mikehaertl) [inline UrlManager.php]: Please point to the section in PSR-2 where this is specified.

File: tests/EventTest.php [hunk 4]
```diff
@@ -97,8 +98,8 @@ public function testFiresNotIfNoSessionLanguageChange()
     public function testFiresNotIfPersistenceDisabled()
     {
         $this->mockUrlManager([
-            'languages' => ['fr', 'en', 'de'],
-            'on languageChanged' => [$this, 'languageChangedHandler'],
+            'languages'                 => ['fr', 'en', 'de'],
+            'on languageChanged'        => [$this, 'languageChangedHandler'],
```
Comment (mikehaertl) [inline tests/EventTest.php]: We don't align `=>` and I don't want to introduce this. (Had this before and it only leads to trouble e.g. when modifying code that requires a change of alignment depths: you won't easily see the actual changed line in git history).

**Comment by thyseus** [comment]
hey @mikehaertl i agree, this PR introduces too much personal flavor. closing the PR :)

**Pull request closed without merging**